### PR TITLE
template literals: changed explanation about illegal escape sequences

### PR DESCRIPTION
**Pull request by HolgerJeromin**
#### Summary
Removed link to fixed issue. 
Instead linked to existing explanation.

#### Supporting details
https://2ality.com/2016/09/template-literal-revision.html
https://github.com/tc39/proposal-template-literal-revision/issues/5
https://github.com/tc39/notes/blob/HEAD/meetings/2017-03/mar-21.md#10ia-template-literal-updates

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks for the changes! I would even go further with the first part.

File: files/en-us/web/javascript/reference/global_objects/string/raw/index.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ browser-compat: javascript.builtins.String.raw
 The static **`String.raw()`** method
 is a tag function of [template literals](/en-US/docs/Web/JavaScript/Reference/Template_literals).
 This is _similar_ to the `r` prefix in Python, or the `@`
-prefix in C# for string literals. (But it is not _identical_; see explanations
-in [this issue](https://bugs.chromium.org/p/v8/issues/detail?id=5016).)
+prefix in C# for string literals. (But it was not _identical_ till ES2018 see
```
[the remark this answers is not in the document]
Comment (HolgerJeromin) [reply files/en-us/web/javascript/reference/global_objects/string/raw/index.md:18]: Fixed 5 years ago in most engines. Yes.
But Microsoft Edge 13-18 do not support this...

[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/string/raw/index.md:18]: Edge 13 - 18 are outdated. Edge 79 was released in early 2019 (Jan.15th), more than 3 years ago, I think we can ignore Edge18 and earlier outside the browser compat table.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

**Pull request merged**